app/cdap/components/GridTable/components/NestedMenu/menuOptions/datatypeOptions.ts

```diff
@@ -21,6 +21,28 @@ export const DATATYPE_OPTIONS = [
     label: T.translate(
       'features.WranglerNewUI.GridPage.transformations.options.labels.datatype.string'
     ).toString(),
+    options: [
+      {
+        value: 'boolean',
+        label: T.translate(
+          'features.WranglerNewUI.GridPage.transformations.options.labels.datatype.boolean'
```

Comment:
plz move the common path to PREFIX variable inside the constants.ts file
and reuse from there
we can just change the green code for now

we have references in google's old code
plz have a look

Reply:
Done